packages/components/src/components/popover/popover.jsx

```diff
@@ -70,6 +70,7 @@ class Popover extends React.PureComponent {
             message,
             zIndex,
             relative_render,
+            should_pointer_event_disabled,
```

Comment:
The wording seems unnatural, could we rename this to something like below:

```suggestion
            should_disable_pointer_events,
```